akka-stream/src/main/boilerplate/akka/stream/javadsl/UnzipWith.scala.template

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.javadsl
+
+import akka.stream._
+import akka.stream.scaladsl
+import akka.japi.function
+import akka.japi.tuple
+import akka.japi
+
+/**
+ * Split one stream into several streams using a splitting function.
+ *
+ * '''Emits when''' all of the outputs stops backpressuring and there is an input element available
+ *
+ * '''Backpressures when''' any of the outputs backpressures
+ *
+ * '''Completes when''' upstream completes
+ *
+ * '''Cancels when''' any downstream cancels
+ */
+object UnzipWith {
+
+  /**
+   * Create a new `UnzipWith` vertex with the specified input type and unzipping-function `f`.
+   *
+   * @param f unzipping-function from the input value to the pair of output values
+   */
+  def createWithPair[In, A, B](f: function.Function[In, japi.Pair[A, B]]):  Graph[FanOutShape2[In, A, B], Unit] =
```

Comment:
I'm not sure about `createWithPair` hm...

Since this PR adds Tuples to japi, we now have `Pair` *and* `Tuple2` for javadsl, we should probably remove one of them. // cc @akka/akka-team